docs/src/main/asciidoc/security-authentication-mechanisms.adoc

```diff
@@ -386,12 +393,15 @@ For example, it can be a public endpoint or be protected with mTLS.
 In this scenario, you do not need to protect your Quarkus endpoint by using the Quarkus OpenID Connect adapter.
 ====
 
+ifndef::no-quarkus-oidc-token-propagation[]
```

Comment:
@rolfedh Have we also agreed to block the dev-preview content ?

Reply:
Unfortunately, this extension [is listed as unsupported in 3.8](https://code.quarkus.redhat.com/?extension-search=origin:platform%20quarkus-oidc-token-propagation).

Reply:
At the same time, we have a release note that doesn't mention the extension specifically but says this:

```
1.3.4.12. OIDC token propagation supported
Red Hat build of Quarkus 3.8 introduces support for OIDC token propagation.

With this update, Red Hat build of Quarkus endpoints use REST clients to propagate the incoming OIDC access tokens to other secure endpoints that expect access tokens.
```
Can you help me understand?

Reply:
The three sources of information must agree with each other:
- Guides
- Release notes
- https://code.quarkus.redhat.com/

Reply:
@rolfedh Sounds like we need to remove this section from the release notes or update it to say that it is in `Dev Preview`, @rsvoboda, @maxandersen, what should be the preferred solution here ? thanks

Reply:
@maxandersen @rsvoboda please comment.

TLDR: There is a discrepancy regarding the OIDC token propagation extension, `quarkus-oidc-token-propagation`, in the Red Hat build of Quarkus 3.8:

- Release Note Ambiguity: This [OIDC token propagation supported](https://access.redhat.com/documentation/en-us/red_hat_build_of_quarkus/3.8/html-single/release_notes_for_red_hat_build_of_quarkus_3.8/index#oidc-token-propagation-supported) release note in 3.8 mentions support for OIDC token propagation but does not specify if it is in DEV-PREVIEW.
- Website Discrepancy: The extension does not have status labels indicating that it is supported [here at https://code.quarkus.redhat.com](https://code.quarkus.redhat.com/?extension-search=origin:platform%20quarkus-oidc-token-propagation). If it is supported, someone needs to update the labels. 
- Documentation:  If the extension is supported, I will remove or turn off the conditionalization so that the corresponding content appears in the product documentation.

Reply:
https://issues.redhat.com/browse/QUARKUS-4179 - this is planned in 3.8.z stream, @tqvarnst do you want it for 3.8.5? If so it's another aspect to consider in the planning of that release.

Reply:
@maxandersen @sberyozkin @rsvoboda I'd like to move ahead with getting this PR approved and merged.
The question of when to show or hide `quarkus-oidc-token-propagation`-related content can be handled in the product attributes file and shouldn't be a blocker for this PR.

Reply:
Merging this PR will enable me to migrate the files to the product repo so I can build previews for QE to review.

Reply:
I think we can have quarkus-oidc-token-propagation stuff in as we plan to support it starting with 3.8.5